clang/lib/Tooling/Core/Replacement.cpp

```diff
@@ -342,6 +342,20 @@ llvm::Error Replacements::add(const Replacement &R) {
   return llvm::Error::success();
 }
 
+llvm::Error Replacements::addOrMerge(const Replacement &R) {
+  auto Err = add(R);
+  if (Err) {
```

Comment:
nit: You could rewrite this as:
```suggestion
  if (llvm::Error Err = add(R)) {
```